packages/jseql/src/ffi/index.ts

```diff
@@ -1,267 +1,417 @@
 import {
   newClient,
-  decrypt,
-  encrypt,
-  encryptBulk,
-  decryptBulk,
+  encrypt as ffiEncrypt,
+  decrypt as ffiDecrypt,
+  encryptBulk as ffiEncryptBulk,
+  decryptBulk as ffiDecryptBulk,
 } from '@cipherstash/jseql-ffi'
 import { logger } from '../../../utils/logger'
-import type { LockContext } from '../identify'
 import { checkEnvironmentVariables } from './env-check'
 import {
-  normalizeBulkDecryptPayloads,
   normalizeBulkEncryptPayloads,
+  normalizeBulkDecryptPayloads,
 } from './payload-helpers'
+import type { LockContext } from '../identify'
+
+// ------------------------
+// Type Definitions
+// ------------------------
+export type EncryptPayload = string | null
+
+export type EncryptedPayload = { c: string } | null
+
+export type BulkEncryptPayload = {
+  plaintext: string
```

Comment:
Should this be `EncryptPayload` instead of `string`? As in a bulk operation is essentially an array of single ops along with an id?

Reply:
No - due to https://github.com/cipherstash/jseql/issues/42